Phishing enhancements spear (#27062)
demisto/content: Packs/Phishing/Playbooks/playbook-Spear_Phishing_Investigation_README.md

> The "Spear Phishing Investigation" Playbook designed to detect patterns that indicates a spear phishing attempt by the attacker.

Reviewer: ```suggestion
The "Spear Phishing Investigation" playbook is designed to detect patterns that indicates a spear phishing attempt by the attacker.
```

## Dependencies

This playbook uses the following sub-playbooks, integrations, and scripts.

### Sub-playbooks

This playbook does not use any sub-playbooks.

### Integrations

This playbook does not use any integrations.

### Scripts

> * MatchRegexV2

Reviewer: ```suggestion
MatchRegexV2
```

### Commands

> * setIncident

Reviewer: ```suggestion
setIncident
```

## Playbook Inputs

---

| **Name** | **Description** | **Default Value** | **Required** |
| --- | --- | --- | --- |

> | DomainSquattingResults | Results of the domain squatting investigation |  | Optional |

Reviewer: ```suggestion
| DomainSquattingResults | Results of the domain squatting investigation. |  | Optional |
```

> | KeywordsToSearch | A comma-separated or list of keywords to search in the email body.<br/>for example: name of the organization finance app that the attacker might impost to. |  | Optional |

Reviewer: ```suggestion
| KeywordsToSearch | A comma-separated list of keywords to search in the email body.<br/>For example: name of the organization finance app that the attacker might impersonate. |  | Optional |
```
